module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -169,6 +169,51 @@ Read the full documentation and download cheatsheets (where available) for these
 
 ### `Seurat` and `SCE` object conversion
 
+This [vignette](https://satijalab.org/seurat/articles/conversion_vignette.html) from the `Seurat` documentation explains how to convert between `SCE` and `Seurat` formats.
+
+In addition, we provide some code examples below for how you can accomplish these conversions.
+For all code examples below, it is assumed that `SingleCellExperiment` and `Seurat` libraries have been loaded in your R environment:
+
+```r
+library(Seurat)
+library(SingleCellExperiment)
+```
+
+
 #### Converting from `Seurat` to `SCE`
 
+The following example code assumes you have a `Seurat` object called `seurat_obj`.
+
+```r
+# Convert Seurat object to SCE object using the
+#  Seurat function `as.SingleCellExperiment()
+sce_object <- as.SingleCellExperiment(seurat_obj)
+```
+
+Alternatively, you can extract individual slots from the `Seurat` object and build your `SCE` object from scratch.
+You can refer to the ["The `SingleCellExperiment` class" chapter in _Orchestrating Single Cell Analyses_](http://bioconductor.org/books/release/OSCA.intro/the-singlecellexperiment-class.html) for approaches to constructing `SCE` objects.
+For example, to obtain an SCE object that contains only the raw counts stored in the `Seurat` object's "RNA" assay:
+
+```r
+# First, extract the counts matrix from the Seurat object
+counts_matrix <- GetAssayData(seurat_obj[["RNA"]])
+
+# Create an SCE object from the counts matrix
+sce_object <- SingleCellExperiment(assays = list(counts = counts_matrix))
+```
+
 #### Converting from `SCE` to `Seurat`
+
+This [documentation from the `ScPCA`](https://scpca.readthedocs.io/en/latest/faq.html#what-if-i-want-to-use-seurat-instead-of-bioconductor) introduces how to convert `SCE` objects to `Seurat` objects.
+Although this documentation was written for `ScPCA` datasets, the steps generally apply to any `SCE object`.
+Briefly, here is how you can convert a `Seurat` to `SCE` object, focusing on porting over _assays_.
+We provide `"RNA"` as the argument to `assay`, as this `Seurat`'s default name for raw count matrices.
+```r
+# Create seurat object from the existing `sce_object`'s counts matrix,
+#  using the Seurat function `CreateSeuratObject`
+seurat_object <- CreateSeuratObject(counts = counts(sce_object),
+                                    assay = "RNA",
+                                    project = "name of your project goes here")
+```
+
```

Comment:
We may want to note that we are not using `Seurat::as.Seurat()` though that may be an option (it didn't used to work very well, but maybe works better now?)

Reply:
I'll check it out on whatever versions we have in the training `renv`, haven't use that function yet.